simp_le.py

```diff
@@ -243,8 +266,9 @@ class IOPlugin(object):
     - for `cert`: certificate, an instance of `OpenSSL.crypto.X509`
     - for `chain`: certificate chain, a list of `OpenSSL.crypto.X509` instances
     """
+    Data.__new__.__defaults__ = (None,) * 4
```

Comment:
I found this approach on SO, here is one for example: 
https://stackoverflow.com/questions/11351032/named-tuple-and-optional-keyword-arguments

The  [Py2 Data model](https://docs.python.org/2/reference/datamodel.html) and [Py3 Data model](https://docs.python.org/3/reference/datamodel.html) docs allow this:

> **Attribute**: `__defaults__`  
> **Meaning**: A tuple containing default argument values for those arguments that have defaults, or None if no arguments have a default value  
> Writable